ajax/libs/sql-formatter/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "sql-formatter",
+  "version": "2.3.1",
+  "description": "Formats whitespaces in a SQL query to make it more readable",
+  "license": "MIT",
+  "main": "dist/sql-formatter.min.js",
```

Comment:
Let's modify as `"filename": "sql-formatter.min.js",`
This is the main file of this library, you might be able to find it after reading its documentation.
Thanks